packages/graphql-lowcode/__tests__/testScenarios.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * No fragment in query
+ */
+export const noFragmentQuery = 
+`{
+  field1
+  field2 {
+    field3
+  }
+}`
+
+/**
+ * Different type query
+ */
+
+export const diffTypeQuery = 
+`{
```

Comment:
please, add few more test scenarios please with test data that also contain query like this:
```
query x {
  ...fragmentName
}
fragment fragmentName on diffEntity {
    field1
}`
```

we will ensure that it will work also for real worlds scenarios